data/mods/Magiclysm/items/ammo_types.json

```diff
@@ -16,5 +16,11 @@
     "id": "blood",
     "name": "blood",
     "default": "blood"
+  },
+  {
+    "type": "ammunition_type",
+    "id": "arrow_orichalcum",
+    "name": "orichalcum arrow",
+    "default": "arrow_orichalcum"
```

Comment:
Please don't duplicate ids (you have the same id for arrows and this ammotype).
I believe it's allowed, but it's very confusing.

Reply:
Fixed that, my fault I assumed it needed to be like that. I know for future reference on the ammo type.